x-pack/plugins/alerts/server/alerts_client/alerts_client.ts

```diff
@@ -236,6 +236,8 @@ export class AlertsClient {
       throw error;
     }
 
+    this.alertTypeRegistry.ensureAlertTypeEnabled(data.alertTypeId);
+
     // Throws an error if alert type isn't registered
     const alertType = this.alertTypeRegistry.get(data.alertTypeId);
```

Comment:
Since `ensureAlertTypeEnabled` calls `get` internally, we can remove the `.get` call while keeping the comment.
```suggestion
    // Throws an error if alert type isn't registered
    this.alertTypeRegistry.ensureAlertTypeEnabled(data.alertTypeId);
```

Reply:
But alertType is used in the code below in the create method..

Reply:
Oh, I see 🙈 all good then.